### PR TITLE
Feature/ufs nst

### PR DESCRIPTION
**Pull request by XuLi-NOAA**
One NSST model constant change and the inclusion of the ccpp/physics submodule change #219, see https://github.com/NOAA-EMC/fv3atm/issues/219. 

This change has impact on the NSST model and ccpp/physics (if the NSST model is on) 

The new baseline results are at: /scratch1/NCEPDEV/stmp4/Xu.Li/FV3_RT/REGRESSION_TEST_INTEL

Dependencies:
ufs-weather-model PR[#483](https://github.com/ufs-community/ufs-weather-model/pull/483)
ccpp physics PR[#589](https://github.com/NCAR/ccpp-physics/pull/589)

### CODE REVIEW

**Comment by junwang-noaa** [comment]
Add reviewers whom the author suggested.

**Review by SMoorthi-emc — APPROVED** [review]

File: ccpp/suites/suite_FV3_GFS_v15p2_couplednsst.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (climbfuji) [inline ccpp/suites/suite_FV3_GFS_v15p2_couplednsst.xml:1]: Do we still need a GFS v15p2 coupled nsst suite, given that we are moving all tests over to GFS v16 ?

Comment (climbfuji) [reply ccpp/suites/suite_FV3_GFS_v15p2_couplednsst.xml:1]: In fact, this suite isn't used in the regression tests - remove please.

**Comment by SMoorthi-emc** [comment]
So it is now the policy that only suites used in regression tests are saved?
I have several suites that are not in the regression tests and I can't add them to repository?

**Comment by climbfuji** [comment]
> So it is now the policy that only suites used in regression tests are saved?
> I have several suites that are not in the regression tests and I can't add them to repository?

No, please don't misunderstand me. If there is a good reason for having that suite, then of course it is ok to keep it. Sorry if my comment was misleading. But we do need to keep an eye on the number (and proliferation) of suites in that directory.

**Comment by XuLi-NOAA** [comment]
The suite

suite_FV3_GFS_v15p2_couplednsst.xml
<https://github.com/NOAA-EMC/fv3atm/pull/255#discussion_r607857912>

has been used in the evaluation of the NSST model with 56 35-day UFS S2S
runs.

It was included at the beginning, but Jun & Bin suggested to add a
test, cpld_bmarkfrac_v16_nsst, which tests both the new suite,
FV3_GFS_v16_couplednsst and NSST model in the coupled model, therefore,
FV3_GFS_v15p2_couplednsst is not in the tests anymore.

And, it is good to keep it for a while and can be removed later on with
more updates.

**Review by climbfuji — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]

**Comment by binli2337** [comment]
@junwang-noaa @DusanJovic-NOAA Ready to merge. Thanks!

**Comment by climbfuji** [comment]
This merge was incorrect - wrong ccpp-physics hash. Need to fix with a follow-up PR before ufs-weather-model can be updated and mergedd.

**Pull request merged**